libs/barista-components/sunburst-chart/README.md

```diff
@@ -0,0 +1,104 @@
+# SunburstChart
+
+<ba-ux-snippet name="sunburst-chart-intro"></ba-ux-snippet>
+<ba-live-example name="DtExampleSunburstChartDefault" fullwidth></ba-live-example>
+
+## Imports
+
+You have to import the `DtSunburstChartModule` when you want to use the
+`<dt-sunburst-chart>`:
+
+```typescript
+@NgModule({
+  imports: [DtSunburstChartModule],
+})
+class MyModule {}
+```
+
+## Example setup
+
+<ba-live-example name="DtExampleSunburstChartRelativeValues" fullwidth></ba-live-example>
+
+## Initialization
+
+To create a `dtSunburstChart` in a minimal configuration, only `series`
+attribute is required to create a valid output.
+
+<ba-live-example name="DtExampleSunburstChartDefault" fullwidth></ba-live-example>
+
+## Options & Properties
+
+### DtSunburstChart
+
+#### Inputs
+
+| Name               | Type                     | Default      | Description                                                     |
+| ------------------ | ------------------------ | ------------ | --------------------------------------------------------------- |
+| `series`           | `DtSunburstChartNode[]`  | -            | Array of nodes with their children (i.e [A,B,C]).               |
+| `selected`         | `DtSunburstChartNode[]`  | -            | Array of selected nodes (i.e [A, A.1, A.1.a]).                  |
+| `noSelectionLabel` | `string`                 | `All`        | Label to be shown when there's no selection.                    |
+| `valueDisplayMode` | `'absolute' | 'percent'` | `'absolute'` | Mode of value visualization. It can be `percent` or `absolute`. |
+
+#### Outputs
+
+| Name             | Type                                  | Description                                                                                    |
+| ---------------- | ------------------------------------- | ---------------------------------------------------------------------------------------------- |
+| `selectedChange` | `EventEmitter<DtSunburstChartNode[]>` | Event that fires when a node is clicked with an array of selected nodes (i.e [A, A.1, A.1.a]). |
+
+### DtSunburstChartOverlay
+
+The `dtSunburstChartOverlay` directive applies to an `ng-template` element lets
+you provide a template for the rendered overlay. The overlay will be shown when
+a user hovers the slice in sunburst-chart. `tooltip` is a
+`DtSunburstChartExtendedNode`.
+
+```html
+<ng-template dtSunburstChartOverlay let-tooltip>
+  <!-- Insert your template for one event here. -->
+</ng-template>
+```
+
+## DtSunburstChartNode
+
+This `DtSunburstChartNode` can be extended to get as much information as needed
+(i.e. filterName, filterValue)
```

Comment:
I think this sentence is more focused on the developer that implements the component rather than the user. 
```suggestion
This `DtSunburstChartNode` holds the information for the slices in the sunburst chart.
```